Configuration/AlCa/python/autoCond.py

```diff
@@ -24,11 +24,11 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run2
     'run2_mc_pa'                   : '123X_mcRun2_pA_v1',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'                    : '123X_dataRun2_v3',
+    'run2_data'                    : '124X_dataRun2_Candidate_2022_06_06_10_25_34',
```

Comment:
@cms-sw/alca-l2 please provide suitable GTs

Reply:
Will do. Do you prefer if we give it to you, or have a separate dedicated PR?

Reply:
I would prefer to have it self-consistent.